Code/Commando/dlgmovieoptions.cpp

```diff
@@ -81,7 +81,7 @@ MovieOptionsMenuClass::On_Init_Dialog (void)
 		RegistryClass registry (APPLICATION_SUB_KEY_NAME_MOVIES);
 		if (registry.Is_Valid ()) {
 
-			const char *INTRO_MOVIE	= "DATA\\MOVIES\\R_INTRO.BIK";
+			const char *INTRO_MOVIE	= "MOVIES\\R_INTRO.BIK";
```

Comment:
What's the reason for this change? The working directory of renegade is the directory containg `DATA`. So `data/movies/r_intro.bik` should work.